Update Configuration:-Key-Bindings.md (#893)
YaLTeR/niri: wiki/Configuration:-Key-Bindings.md

- `Alt`;
- `Super` or `Win`;
- `ISO_Level3_Shift` or `Mod5`—this is the AltGr key on certain layouts;

> - `ISO_Level5_Shift` used by setting xkb lv5 option (check xkb/rules/base.lst)

Reviewer: ```suggestion
- `ISO_Level5_Shift`: can be used with an xkb lv5 option like `lv5:caps_switch`;
```
